docs/pages/database-access/reference/configuration.mdx

```diff
@@ -111,6 +111,12 @@ spec:
     ...
     -----END CERTIFICATE-----
 
+  # MySQL only options.
+  mysql:
+    # The default MySQL server version reported by the Teleport Proxy Service.
```

Comment:
```suggestion
    # The MySQL server version reported by the Teleport Proxy Service.
    # Teleport uses this string when reporting the server version to a 
    # connecting client.
    #
    # When this option is not set, the Database Service will try to connect to
    # a MySQL instance on startup and fetch the server version. Otherwise, 
    # it will use the provided value without connecting to a database. 
    #
    # In both cases, the MySQL server version reported to a client will be 
    # updated on the first successful connection made by a user.
```